src/ipc-handlers.ts

```diff
@@ -1081,6 +1070,95 @@ END` );
 	}
 }
 
+export async function openAppAtPath(
+	_event: IpcMainInvokeEvent,
+	editorKey: SupportedEditor,
+	filePath: string
+): Promise< void > {
+	const platform = process.platform;
+	const editor = supportedEditorConfig[ editorKey ];
+
+	if ( ! editor ) {
+		throw new Error( `Editor ${ editorKey } not found or not supported` );
+	}
+
```

Comment:
```suggestion
```
Typescript already guarantees that `editor` is a `SupportedEditorConfig` object.

Reply:
Thanks, done in a6824001.